### PR TITLE
Code Fix for CI run

### PR DESCRIPTION
**Pull request by kaushik691**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -114,17 +110,33 @@ jobs:
...
           
           mkdir -p backup
-          tree
+          #tree
           cat test_suites/dgraph/Linux/dgraph_suite_01.robot
-          export PATH=~/go/bin/:$PATH
-          echo " ---- Running Dgraph Testcases ----"
-          robot --splitlog -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_01.robot
+          echo " ---- Running Dgraph Testcases 01 ----"
+          #sudo robot --splitlog -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_01.robot
```
Comment (souravDgraph) [inline .github/workflows/nightly.yml:116]: uncomment

File: .github/workflows/weeklyLargeData.yml [hunk 2]
```diff
@@ -88,13 +90,17 @@ jobs:
           tree
           cat test_suites/dgraph/Linux/dgraph_suite_01.robot
           echo " ---- Running Dgraph Testcases ----"
-          robot --splitlog --include WEEKLY -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_01.robot
-          robot --splitlog --include WEEKLY -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_03.robot
+          sudo robot --splitlog --include WEEKLY -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_01.robot
+          sudo robot --splitlog --include WEEKLY -d results/ --loglevel TRACE:INFO test_suites/dgraph/Linux/dgraph_suite_03.robot
```
Comment (souravDgraph) [inline .github/workflows/weeklyLargeData.yml:94]: comment this.

File: lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py [hunk 3]
```diff
@@ -189,10 +208,10 @@ def backup_using_admin(self, path):
                                              variables=variables, cert=cert)
         except Exception as err:
             raise Exception(f"Something went wrong with the backup request.. {json.dumps(response.json())}") from err
-        assert_equal(response.json()['data']['backup']['response']['code'], 'Success')
-        assert_equal(response.json()['data']['backup']['response']['message'], 'Backup completed.')
-        logger.info('Backup successfully completed')
-        return response
+        # assert_equal(response.json()['data']['backup']['response']['code'], 'Success')
```
Comment (souravDgraph) [inline lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py:211]: are these asserts not required?

File: lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py [hunk 4]
```diff
@@ -248,10 +267,10 @@ def backup_using_admin_backup(self, path):
             response = self.req_handler.post_request(appender=appender, headers=headers, body=payload, cert=cert)
         except Exception as err:
             raise Exception(f"Something went wrong with the backup request... {json.dumps(response.json())}") from err
-        assert_equal(response.json()['data']['backup']['response']['code'], 'Success')
-        assert_equal(response.json()['data']['backup']['response']['message'], 'Backup completed.')
+        # assert_equal(response.json()['data']['backup']['response']['code'], 'Success')
```
Comment (souravDgraph) [inline lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py:270]: same her. Incase not needed remove them.

File: resources/dgraph_commands.robot [hunk 5]
```diff
@@ -528,3 +635,9 @@ Check if parallel process is triggered
     Run Keyword If  '${result_check}'=='True'  Run Keywords     Sleep   30s
     ...     AND     Trigger Loader Process     ${loader_alias}     ${rdf_filename}    ${schema_filename}   ${loader_name}
     ...     AND     Check if parallel process is triggered      ${loader_alias}     ${rdf_filename}    ${schema_filename}   ${loader_name}
+
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:638]: remove extra space.

**Review by souravDgraph — CHANGES_REQUESTED** [review]
Check the YAMLs files once again, both should be almost same. And comment all ludicrous mode changes.

File: test_suites/dgraph/Linux/dgraph_suite_01.robot [hunk 6]
```diff
@@ -56,5 +56,5 @@ TC_06 Perform Increment backup and restore data
 TC_07 Perform parallel live loads.
      [Documentation]    Perform live load data.
      ...    *Author*: Sourav
-     [Tags]    regression   CI
```
[the remark this answers is not in the document]
Comment (kaushik691) [reply test_suites/dgraph/Linux/dgraph_suite_01.robot]: okay

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (kaushik691) [reply .github/workflows/nightly.yml]: done

File: lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py (hunk 3 above)
Comment (kaushik691) [reply lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py:211]: this I'm verifying in dgraph command.robot so for now i have commented this

File: lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py (hunk 4 above)
Comment (kaushik691) [reply lib/dgraph_lib/Dgraph/keywords/custom_request_keywords.py:270]: for now let it be.. will remove after few runs

File: resources/dgraph_commands.robot (hunk 5 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:638]: okay

**Review by souravDgraph — APPROVED** [review]

**Pull request merged**